sdk/src/main/java/io/radar/sdk/RadarActivityLifecycleCallbacks.kt

```diff
@@ -52,6 +52,7 @@ internal class RadarActivityLifecycleCallbacks(
                     val usage = "resume"
                     Radar.apiClient.getConfig(usage, false, object : RadarApiClient.RadarGetConfigApiCallback {
                         override fun onComplete(config: RadarConfig) {
+                            Radar.warmupStandardIntegrityTokenProvider(config)
```

Comment:
@tderouin Is there a need to expose this publicly?

Reply:
I'm not sure what you mean. Can you clarify? I'd like to warm up the token provider on `onActivityResumed`. Is there a better way to do this?